design/one-pager-resource-status.md

```diff
@@ -0,0 +1,133 @@
+# Resource High Level Stop-Light Status
+
+- Owner: Steven Rathbauer ([@rathpc](https://github.com/rathpc))
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Problem
+
+As a stack author there is currently no way to author rules that combine arbitrary conditions of a given state to
+represent high level _"stop-light"_ conditions.
+
+The high level conditions would be:
+- Online
+- Offline
+- Warning
+- Unknown
+
+The **Unknown** condition is interesting because this would be returned if the conditions were annotated but we cannot
```

Comment:
Additional context on what it means to be Online, Offline, or in a Warning state would be helpful, especially because this context has been provided for Unknown.

While Online/Offline may appear obvious, there may be nuance in the description.  (This might even compel a rewording to Available/Unavailable).

Reply:
Just updated 👍